examples/python/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "port": "8080",
+    "root_url": "https://6fe6-103-157-160-168.in.ngrok.io",
+    "host": "0.0.0.0"
+}
```

Comment:
I think it's confusing how there are 3 sources of configuration (hardcode, settings file, and env vars). I'm thinking we should get rid of this file, as it's not something established among the examples already